app/views/providers/users/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= content_for :page_title, "Users" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Users</h1>
+    <p class="govuk-body">
+      These users currently have access to this account. They can manage all your course and organisation details, including publishing, closing and withdrawing courses.
+    </p>
+    <%= link_to "Invite user", request_access_provider_path(code: @provider.provider_code), class: "govuk-button" %>
+    <% @users.each do |user| %>
+      <div>
+        <h2 class="govuk-heading-m govuk-!-margin-bottom-1">
+          <%= user.first_name user.last_name %>
+        </h2>
+        <p class="govuk-body govuk-!-margin-bottom-0"><%= user.email %></p>
+      </div>
```

Comment:
I'm not sure each users name has to be a heading.
also it might be better to make it a list of users and style them to look like different "sections"

Reply:
It's consistent with APPLY - https://manage-applications-beta.herokuapp.com/users

Reply:
I'm not suggesting we should change the visual style just the markup. I'm seeing this as a list of users. I don't think theres enough content exposed about the user from one or more sources to warrant it to be considered as a "Section" on the page.

Also we should add some context about what the link is for. The context could be initially visually hidden for screenreader but it would also benefit visually. Without clicking on the Link I have no idea what is going to happen if I click the link. Will I email the user (theres a mail address at the bottom which gives this impression)? Would it activate/delete or edit?